src/main/java/seedu/address/logic/Messages.java

```diff
@@ -18,8 +18,13 @@ public class Messages {
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d people listed!";
     public static final String MESSAGE_DUPLICATE_FIELDS =
                 "Multiple values specified for the following single-valued field(s): ";
-    public static final String MESSAGE_INVALID_TELEGRAM = "Cannot find person with telegram handle: %1$s";
+    public static final String MESSAGE_INVALID_TELEGRAM = "Cannot find person with telegram handle: @";
+    public static final String MESSAGE_TOO_SHORT_TELEGRAM =
```

Comment:
I believe ziyuan will restrict telegram length to [5, 32] characters in his new pr. Also maybe you wanna import the validation code from his Telegram class instead so that it will be more consistent